packages/rol/adapters/thyra/src/function/ROL_ThyraProductME_Constraint_SimOpt.hpp

```diff
@@ -61,16 +62,22 @@ class ThyraProductME_Constraint_SimOpt : public Constraint_SimOpt<Real> {
 
 public:
 
-  ThyraProductME_Constraint_SimOpt(Thyra::ModelEvaluatorDefaultBase<double>& thyra_model_, int g_index_, const std::vector<int>& p_indices_,Teuchos::RCP<Teuchos::ParameterList> params_ = Teuchos::null, bool recompute = false) :
-    thyra_model(thyra_model_), g_index(g_index_), p_indices(p_indices_), params(params_) {
+  ThyraProductME_Constraint_SimOpt(Thyra::ModelEvaluatorDefaultBase<double>& thyra_model_, int g_index_, const std::vector<int>& p_indices_,
+    Teuchos::RCP<Teuchos::ParameterList> params_ = Teuchos::null, Teuchos::EVerbosityLevel verbLevel= Teuchos::VERB_HIGH) :
```

Comment:
Should the default verbosity level be high?

Reply:
I was debating about this. I thought that when one tries it out, without knowing all the options, it's good to have more info in output. For real applications, one can change the verbose level as preferred. Anyway, it's not going to affect performance, it's just many print outs (I'm not printing out, e.g. the entries of a vector). I can change if you disagree with this.